src/main/java/br/com/studiesMaterials/handlers/CourseHandler.java

```diff
@@ -0,0 +1,103 @@
+package br.com.studiesMaterials.handlers;
+
+import br.com.studiesMaterials.dao.CourseDao;
+import br.com.studiesMaterials.db.DataBase;
+import br.com.studiesMaterials.web.api.schemas.CourseSchema;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+import com.google.gson.Gson;
+
+import java.sql.*;
+
+
+public class CourseHandler implements CourseDao {
+    @Override
+    public APIGatewayProxyResponseEvent create(APIGatewayProxyRequestEvent input) {
+        APIGatewayProxyResponseEvent responseEvent = new APIGatewayProxyResponseEvent();
+        try(Connection conn = DataBase.connection()) {
+            assert conn != null;
+            Statement statement = conn.createStatement();
+
+            String studentId =  input.getPathParameters().get("student_id");
+
+            CourseSchema data = convertBody(input.getBody());
+
+            String sql = String.format(
+                    "INSERT INTO public.course (student_id, name, platform, price) " +
+                            "VALUES ('%s', '%s', '%s', '%s')", studentId, data.name, data.platform, data.price
+            );
+
+            statement.executeUpdate(sql);
+
+            responseEvent.setStatusCode(201);
+
+            return responseEvent;
+        } catch (SQLException error) {
+            error.printStackTrace();
+            responseEvent.setStatusCode(500);
+
+            return responseEvent;
+        }
+    }
+
+    @Override
+    public APIGatewayProxyResponseEvent update(APIGatewayProxyRequestEvent input) {
+        APIGatewayProxyResponseEvent responseEvent = new APIGatewayProxyResponseEvent();
+        try(Connection conn = DataBase.connection()) {
+            assert conn != null;
+            Statement statement = conn.createStatement();
+
+            String courseId = input.getPathParameters().get("course_id");
+
+            CourseSchema data = convertBody(input.getBody());
+
+            String sql = String.format(
+                    "UPDATE public.course SET name = '%s', platform = '%s', price = '%s'" +
```

Comment:
Precisa adicionar o espaço depois de fechar a aspa simples, porque senão vai juntar o valor da variável com o 'where'.
```suggestion
                    "UPDATE public.course SET name = '%s', platform = '%s', price = '%s' " +
```